Reexport Web3 objects after module augmentation and update README.md with example (#4)
ChainSafe/web3.js-plugin-chainlink: README.md

yarn add @chainsafe/web3.js-chainlink-plugin
```

## Using this plugin

You can use this plugin to get the price of token-pairs from a Chainlink smart contract.

> And here is a code snippet that get the Price of the pair `Link/Eth` from the Ethereum mainnet.

Reviewer: ```suggestion
Following is a code snippet is for getting Price of the pair `Link/Eth` from the Ethereum mainnet.
```

> However, if you are connected to another Ethereum network other than the mainnet, or you would like to get the price of another pair, replace `MainnetPriceFeeds.LinkEth`, in the code below, with the address of the intended Chainlink smart contract that is deployed on the network that you are connected to. And you may check https://docs.chain.link/docs/data-feeds/price-feeds/addresses/ for the most updated mainnet and testnet addresses.

Reviewer: ```suggestion
However, if you are connected to another Ethereum network other than the mainnet, or you would like to get the price of another pair, replace `MainnetPriceFeeds.LinkEth`, in the code below, with the address of the intended Chainlink smart contract that is deployed on the network that you are connected to, and you may check https://docs.chain.link/docs/data-feeds/price-feeds/addresses/ for the most updated mainnet and testnet addresses.
```

> However, if you are connected to another ethereum network other than the mainnet, or you would like to get the price of another pair, replace `MainnetPriceFeeds.LinkEth`, in the code bellow, with the address of the intended Chainlink smart contract that is deployed on the network that you are connected to. And you may check https://docs.chain.link/docs/data-feeds/price-feeds/addresses/ for the most updated mainnet and testnet addresses.

Reviewer 2: ```suggestion
However, if you are connected to another Ethereum network other than the mainnet, or you would like to get the price of another pair, replace `MainnetPriceFeeds.LinkEth`, in the code below, with the address of the intended Chainlink smart contract that is deployed on the network that you are connected to. And you may check https://docs.chain.link/docs/data-feeds/price-feeds/addresses/ for the most updated mainnet and testnet addresses.
```

```typescript
import { ChainlinkPlugin, MainnetPriceFeeds } from '@chainsafe/web3.js-chainlink-plugin';

import Web3 from 'web3';

> const web3 = new Web3('YOUR_ETHEREUM_NODE'); // Use a node connectd to mainnet, if you will keep using `MainnetPriceFeeds.LinkEth` bellow.

Reviewer: ```suggestion
const web3 = new Web3('YOUR_ETHEREUM_NODE'); // Use a node connected to mainnet, if you will keep using `MainnetPriceFeeds.LinkEth` below.
```
